### PR TITLE
Number tasks

### PR DESCRIPTION
**Pull request by NikolayKovzik**
Solved all tasks from the file "numbers-tasks".

### CODE REVIEW

File: js/practice/task/02-numbers-tasks.js [hunk 1]
```diff
@@ -89,7 +89,8 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+    const x = -b / a;
+    return x === -0 ? 0 : x;
```
Comment (echonok) [inline js/practice/task/02-numbers-tasks.js]: why do we compare the result (x) with -0? What is even '-0'?

File: js/practice/task/02-numbers-tasks.js [hunk 2]
```diff
@@ -202,7 +209,16 @@ function roundToPowerOfTen(num, pow) {
...
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    if (n < 2) {
+        return false;
+    }
+    let div = 2;
+    let isNumPrime = true;
+    while (div * div <= n && isNumPrime) {
+        (n % div == 0) ? isNumPrime = false : ++div;
```
Comment (echonok) [inline js/practice/task/02-numbers-tasks.js]: OK, but some people think that it's a bad practice. We use the ternary operator for returning a value, not for anything else
Like in these examples

**Review by echonok — CHANGES_REQUESTED** [review]

File: js/practice/task/02-numbers-tasks.js (hunk 1 above)
Comment (NikolayKovzik) [reply js/practice/task/02-numbers-tasks.js]: I agree, it looks ugly) I couldn't figure out how to get around the test, which crashed due to the comparison of -0 and 0.
Fixed it.

File: js/practice/task/02-numbers-tasks.js (hunk 2 above)
Comment (NikolayKovzik) [reply js/practice/task/02-numbers-tasks.js]: Thanks, fixed

**Review by echonok — APPROVED** [review]

**Pull request merged**